agora-drep/Agora/Proxy.hs

```diff
@@ -186,7 +185,7 @@ proxyScript = plam $ \authSymbol' ctx -> P.do
 
             -- Spending condition 6: Transaction does not include script inputs other than own input.
             PJust ownScriptHash <- pmatch scriptInputs
-            ownCurrencySymbol <- plet $ pscriptHashToCurrencySymbol ownScriptHash
+            let ownCurrencySymbol = pscriptHashToCurrencySymbol ownScriptHash
```

Comment:
This was being somehow magically optimized out, `pscriptHashToCurrencySymbol` is actually just `punsafeCoerce` so I think pletting a var optimization but better to be explicit